addons/adyen_platforms/controllers/adyen_platforms.py

```diff
@@ -1,13 +1,46 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import json
+import logging
+from pprint import pformat
+
 from odoo import http
 from odoo.http import request
 
+from odoo.addons.adyen_platforms.util import odoo_payments_proxy_control
+
+_logger = logging.getLogger(__name__)
+
 
 class AdyenPlatformsController(http.Controller):
 
     @http.route('/adyen_platforms/create_account', type='http', auth='user', website=True)
     def adyen_platforms_create_account(self, creation_token):
         request.session['adyen_creation_token'] = creation_token
         return request.redirect('/web?#action=adyen_platforms.adyen_account_action_create')
+
+    @odoo_payments_proxy_control()
+    @http.route('/adyen_platforms/account_notification', type='json', auth='public', csrf=False)
+    def adyen_platforms_notification(self):
+        data = json.loads(request.httprequest.data)
+        _logger.debug('Account notification received: %s', pformat(data))
+
+        account = request.env['adyen.account'].sudo().search([('adyen_uuid', '=', data.get('adyen_uuid'))])
+        if not account:
+            _logger.error('Received notification for non-existing account')
```

Comment:
1. is that actually useful?
2. if it is, it should contain the relevant account identification information.

You can't assume `debug` will be enabled, so if an `error` notification is useful here it should contain actionable information. Maybe not all the data, but at the very least the `adyen_uuid`.

Reply:
I've added the adyen_uuid in the error.